facebook-common/src/main/java/com/facebook/login/CustomTabLoginMethodHandler.java

```diff
@@ -137,6 +140,16 @@ private String getChromePackage() {
         return null;
     }
 
+    private boolean isCustomTabsCompatibleWithAutofill() {
+        if (!Utility.isAutofillAvailable(loginClient.getActivity())) {
+            return true;
+        }
+        // TODO: currently, Chrome does not support Android Autofill, so always return false.
```

Comment:
Do we have an ETA as to when this block would have to be updated?

Reply:
We'd have to wait until a Chrome release supports yet. I don't know in which release that will happen, but I'll keep a close eye and update it once it does.